WeakAuras/Prototypes.lua

```diff
@@ -2859,6 +2859,7 @@ Private.event_prototypes = {
         AddUnitEventForEvents(result, "player", "UNIT_SPELLCAST_STOP")
         AddUnitEventForEvents(result, "player", "UNIT_SPELLCAST_FAILED")
         AddUnitEventForEvents(result, "player", "UNIT_SPELLCAST_SUCCEEDED")
+        AddUnitEventForEvents(result, false, "CURRENT_SPELL_CAST_CHANGED")
```

Comment:
Can you register this event only for era & wotlk?

Reply:
Sure thing!